src/scout_apm/celery.py

```diff
@@ -7,20 +7,21 @@
 from scout_apm.core.tracked_request import TrackedRequest
 
 
-# TODO: Capture queue.
-# https://stackoverflow.com/questions/22385297/how-to-get-the-queue-in-which-a-task-was-run-celery?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa
-def prerun_callback(sender=None, headers=None, body=None, **kwargs):
-    name = kwargs["task"].name
+def prerun_callback(task=None, **kwargs):
+    tracked_request = TrackedRequest.instance()
+    tracked_request.mark_real_request()
 
-    tr = TrackedRequest.instance()
-    tr.mark_real_request()
-    span = tr.start_span(operation=("Job/" + name))
-    span.tag("queue", "default")
+    span = tracked_request.start_span(operation=("Job/" + task.name))
+    delivery_info = task.request.delivery_info
+    span.tag("is_eager", delivery_info.get("is_eager", False))
+    span.tag("exchange", delivery_info.get("exchange", "unknown"))
+    span.tag("routing_key", delivery_info.get("routing_key", "unknown"))
+    span.tag("queue", delivery_info.get("queue", "unknown"))
 
 
-def postrun_callback(sender=None, headers=None, body=None, **kwargs):
-    tr = TrackedRequest.instance()
-    tr.stop_span()
+def postrun_callback(task=None, **kwargs):
```

Comment:
Changed method signature. Who calls this and from where?

Reply:
It's in the same position as prerun_callback above